run-checks

```diff
@@ -267,6 +267,11 @@ if [ "$STATIC" = 1 ]; then
     fi
 
     # FIXME: re-add staticcheck with a matching version for the used go-version
+
+    if [ -z "${SKIP_TESTS_CHECK-}" ] || [ "$SKIP_TESTS_CHECK" = 0 ]; then
+        echo "Checking tests formatting"
+        python3 ./tests/utils/check-tests.py -d ./tests
```

Comment:
Also, perhaps the file should be named check-test-format or somesuch

Reply:
updated